navigation/components/ImageHeader/styles.js

```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  image: {
+    marginTop: -20,
+    width: '100%',
+    height: 110
+  }
+});
```

Comment:
same as the previous comment but use `headerTitleStyle`

Reply:
same, i need it because on larges resolution the header image truncate